knowledge_repo/app/templates/base.html

```diff
@@ -114,7 +114,7 @@
         name: 'knowledge_posts',
         limit: 10,
         display: function (item) {
-          return item;
+          return item.title + " - " + item.author;
```

Comment:
I think this is a more philosophical thing - I would assume seeing a title is more relevant than seeing a person's name, so I'd rather see as much of the title as possible than see a full name + parts of a title. 

2. A lot of posts have multiple authors, so then you'd only see the authors, which isn't super helpful. 

3. The suggestions are in the form `title - author`, so I think we should be consistent between the suggestion + what is displayed